inference-engine/tools/compile_tool/main.cpp

```diff
@@ -289,7 +321,7 @@ static void processPrecisions(InferenceEngine::CNNNetwork &network,
         for (auto &&layer : network.getInputsInfo()) {
             const auto layerPrecision = layer.second->getPrecision();
             if ((isFloat(layerPrecision) && isFloat(precision)) ||
-                (isFP16(layerPrecision) && isU8(precision))) {
+                (isFloat(layerPrecision) && isU8(precision))) {
```

Comment:
Just wondering why did you add this change?

Reply:
Without it you can't set U8 precision for FP32, since by default input will always have FP32 precision (even for FP16 networks).

Reply:
Original goal of this check was, I believe, that we can set U8 or FP16 precision only for FP32 tensor. If the tensor is INTEGER, we can't do this.

Reply:
Got it. Thanks!